news/2nd-asian-pacific-coq-summer-school

```diff
@@ -1,4 +1,4 @@
-<#def ID>94</#def>
+<#def ID>2nd-asian-pacific-coq-summer-school</#def>
```

Comment:
Why do we need to repeat the file name here? I know it was already like that, but it is becoming more painful so I was wondering what it is used for.

Reply:
This PR is only about renaming a few files, not designing a better pre-processor.

If you think it would be a valuable improvement, please open an issue about `yamlpp` not giving access to the name of the file being processed.